index.html

```diff
@@ -136,13 +136,13 @@ <h4>Error:</h4>
     <div id="lesson-popup" class="popup-overlay">
         <div class="popup-content" onclick="event.stopPropagation()">
             <!-- Prevent the popup from closing when clicking inside -->
-            <button class="close-btn" onclick="closeLessonPopup()">×</button>
+            <button class="close-btn" onclick="window.closeLessonPopup()">×</button>
```

Comment:
das window in onclick kann man glaube ich weglassen (?)

Reply:
ja aber da ich es hier auf window setzte finde ich es so besser
https://github.com/TooManyFiles/TMF-Timetable-Frontend/blob/84d39d657bbf956d4942f75e0225b68a22115b95/src/index/table/generatePopup.js#L131-L134